client/.eslintrc.json

```diff
@@ -0,0 +1,50 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
```

Comment:
Sorry if this wasn't clear before, maybe I should've given you some more information about this, but I think you have added too many ESLint rules that are out of scope of this pull request, and adding ESLint "manually" as a dependency was also probably not needed...
As you may know, we are using [create-react-app (CRA)](https://create-react-app.dev/) which provides a lot of tools and settings already out of the box.
CRA isn't fully configurable unless "ejected", you can read more about this here: https://create-react-app.dev/docs/available-scripts#npm-run-eject but we probably want to avoid that unless absolutely necessary because it also has disadvantages.

For ESLint there is a setting that can be changed and then we can add some additional ESLint settings, that will be added to the settings that are provided by CRA, see https://create-react-app.dev/docs/advanced-configuration

So I think that's the way to go, and to stay within the scope of this PR you should probably only add what's needed, which is the `"sort-imports"` rule and `"sort-keys"`